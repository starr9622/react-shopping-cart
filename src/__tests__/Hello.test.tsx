import * as React from 'react';
import { render } from '@testing-library/react';
import Hello from '../components/Hello';
import '@testing-library/jest-dom/extend-expect';

describe('<Hello />', () => {
  it('shows the props correctly', () => {
    const {getByText} = render(<Hello name="hello~~~!" />);
    expect(getByText('Hello~! hello~~~!')).toBeInTheDocument()
  });
});