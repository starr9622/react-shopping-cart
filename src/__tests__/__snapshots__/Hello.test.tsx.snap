// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hello /> matches snapshot 1`] = `
<div>
  <h1>
    Hello~! 
    hello~~~!
  </h1>
</div>
`;
